syntax = "proto3";

package etlpay;

message IngestRequest {
  string source_name = 1;
  string payload_json = 2;
}

message IngestResponse {
  int32 id = 1;
  string status = 2;
}

service EtlService {
  rpc Ingest(IngestRequest) returns (IngestResponse);
}

